### ISSUE TITLE
eliminate goreleaser?

### ISSUE THREAD

**Issue by slingamn** [ergochat/ergo#820]
goreleaser makes me nervous (large, opaque binary dependency) and we're using a very small subset of its functionality. I could probably fit everything goreleaser provides into a few lines of shell.

**Comment by slingamn** [comment]
goreleaser provides the following:

1. Validates that `git status` is clean
2. Validates that HEAD matches an appropriate tag
3. Validates the version string against the tag?
4. Knows which files to include in the archive
5. Knows which os/arch combinations to build

**Comment by slingamn** [comment]
Meh, the old v0.101.0 binary I have still works with vendored modules, so this isn't a priority.

**Comment by DanielOaks** [comment]
Yeah for some reason doing cross-compilation manually always felt a bit hairy or fragile as well, whereas goreleaser just does it without any worries. If we can go our own way and get it working reliably across, say, macOS and Linux then I'd be pretty happy
